bindings/nostr-js/src/types/contact.rs

```diff
@@ -2,8 +2,9 @@
 // Distributed under the MIT software license
 
 use std::ops::Deref;
+use std::str::FromStr;
 
-use nostr::prelude::*;
+use nostr::prelude::{tag::UncheckedUrl, *};
```

Comment:
This in not directly related to this PR so if you pefer you can skip this and I'll do by myself...

All the things inside of `tag` module should be exported automatically from the `prelude` module so, can you add these lines in the `prelude` module?

```rust
pub use crate::event::builder::*;
pub use crate::event::id::*;
pub use crate::event::kind::*;
pub use crate::event::tag::*;
pub use crate::event::unsigned::*;
```

If you make this change, use just `use nostr::prelude::*;`